### PR TITLE
Added Kamva's card

### PR DESCRIPTION
**Pull request by kamva9697**
Just another new Contributor

### CODE REVIEW

**Comment by sanilborkar** [comment]
Hi. Thank you for taking the time to contribute. You're almost there!
Just one change that needs to be done - You need to paste your card exactly above the line
```
<!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
```
Once these changes are done, you can update this PR with the same and I would be happy to merge your changes then. If you have any doubts, please reply here.

File: index.html [hunk 1]
```diff
@@ -10892,6 +10892,44 @@ <h2>A project for first-time contributions</h2>
                     </div>
                 </div>
                 <!-- ________ Joseph's card END ________  -->
+		    
+		    <!-- ________ Kamva's card START ________  -->
+                <div class="card">
```
Comment (sanilborkar) [inline index.html:10897]: You need to paste your card exactly above the line (which would be towards the end of this file)
```
<!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
```

Comment (kamva9697) [reply index.html:10897]: Thank you for taking your to help! I will do so

**Comment by sanilborkar** [comment]
Thank you for your PR, and it is now merged! If you experienced any problems or confusion when carrying out any part of this exercise, please let us know so that we can fill any gaps, if any.

Appreciate your time and effort in contributing to this project.

**Pull request merged**